src/pkg/packaging-tools/framework.dependency.targets

```diff
@@ -222,6 +227,8 @@
         so in case of conflicts, DLLs are found in the framework being crossgenned.
       -->
       <_crossgenPlatformDirectories Include="%(_filesToCrossGen.RootDir)%(_filesToCrossGen.Directory)" />
+      <!-- We need to include System.Windows.Forms as a platform assembly even if we don't crossgen it.-->
+      <_crossgenPlatformDirectories Include="@(WinFormsPlatformDirectory)" />
```

Comment:
I don't think this line is actually necessary here, only in `CreateCrossGenSymbols`. I'm running a new build now to see if I can remove it.

Reply:
Yeah, new build worked (and still fixed the issue).